README.md

```diff
@@ -212,6 +212,18 @@ sites:
     ignore_error: true
 ```
 
+#### `filename`
+
+Overrides the URL based default filename of the site.
+This example would be named `de-edjopato-api-token-0123456789-action-hack-20the-20planet.html` but instead is saved as `de-edjopato-api-planet-hack.html`.
```

Comment:
Add an empty line in between to have a single first explaining sentence then a more in depth explanation.

```suggestion

Normally the filename is automatically derived from the url.
For the following example it would be something like `de-edjopato-api-token-0123456789-action-hack-20the-20planet.html`.
With the `filename` options it is saved as `de-edjopato-api-planet-hack.html` instead.
```